### DISCUSSION TITLE
Хочу еще Дашу

### DISCUSSION THREAD

**Discussion by Mirantus** [frontendu/themes#4] [Гости]
Слушаю все ваши выпуски с самого начала и уже могу судить, что у вас хорошо получается, а что не очень.
Так вот, самый огонь - это выпуски с гостями совершенно из другой сферы, апофеозом которых был выпуск с Дашей психотерапевтом. Это область, которую придумали вы, которая есть только у вас. Так эксплуатируйте ее!
Приглашайте врачей, поваров, сварщиков, бухгалтеров. Послушаем, что они думают про айти.

**Comment by RinatValiullov** [comment]
Хочу!
